### PR TITLE
Draft EIP: BLS12-381 Deterministic Account Hierarchy

### PR DESCRIPTION
**Pull request by CarlBeek**
This is the proposed standard for BLS12-381 Deterministic Account Hierarchy (Account Paths) for use within Eth2 as well as by the larger blockchain industry. It has existed in its own repo for a while now, but this is a better place to enshrine it as a standard for now.

### CODE REVIEW

File: EIPS/eip-draft_bls_path.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+status: Draft
+type: Standards Track
+category: ERC
+created: 2019-09-30
+requires: [eip-draft_bls_keygen.md](https://github.com/CarlBeek/EIPs/blob/bls_keygen/EIPS/eip-draft_bls_keygen.md)
+---
+
+## Simple Summary
+
+This EIP defines the purpose of a given key, or family thereof, within a tree of keys. When combined with [eip-draft_bls_keygen.md](https://github.com/CarlBeek/EIPs/blob/bls_keygen/EIPS/eip-draft_bls_keygen.md), the combination of a seed and knowledge of the desired purpose of a key is sufficient to determine a key pair.
```
Comment (axic) [inline EIPS/eip-draft_bls_path.md]: I believe these keygen links should point to #2333 ?

**Comment by CarlBeek** [comment]
This is ready for merge after #2333 at which point CI should pass as the desired links will exist.

In the most recent commit I have squashed the commits and changed the `coin_type` [as a result of this suggestion](https://github.com/ethereum/EIPs/issues/2338#issuecomment-551446821) to 3600=60^2.

**Comment by eip-automerger** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - Contains new file EIPS/eip-2334.md

**Review by Oleksandr-prog — COMMENTED** [review]
l

**Pull request merged**